### PR TITLE
Fix search results error

### PR DESCRIPTION
**Pull request by bshyong**
Ran into this error while working on https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2098

Search results need to wait until constants are loaded before they can render

Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2259

### CODE REVIEW

**Comment by U-DON** [comment]
Since constants are being fetched at the highest level (`GiBillApp` container), should there be a loading indicator in the render at that level instead?

**Comment by bshyong** [comment]
@U-DON hmm that would blank out the entire UI right? how do you think about moving it up to the `GiBillApp` vs.  the SearchPage component using the existing indicator? 

```
    if (search.inProgress) {
      searchResults = (
        <div className={resultsClass}>
          {filterButton}
          <LoadingIndicator message="Loading search results..."/>
        </div>
      );
```

**Comment by U-DON** [comment]
I think that would work too. But another case to consider is the profile page, since it also depends on the constants for calculations.

**Comment by bshyong** [comment]
^ moved to top-level GIBillApp component

File: src/js/gi/components/search/SearchResult.jsx [hunk 1]
```diff
@@ -92,7 +92,6 @@ export class SearchResult extends React.Component {
       </div>
     );
   }
-
```
Comment (U-DON) [inline src/js/gi/components/search/SearchResult.jsx]: Super nitpick: if removing this newline at the end of the class, can also remove the one at the top.

**Review by U-DON — APPROVED** [review]
LGTM.

**Pull request merged**